_content/v2-preview/java/_quick-start.md

```diff
@@ -0,0 +1,279 @@
+# Quick Start
+This guide will walk you through setting up a Java project that listens to device activations and messages and responds to every 3rd message.
+
+> This guide assumes the sketch and payload functions of [The Things Uno / Quick Start](/uno/#quick-start), but can be easily applied to any other.
+
+The full script that we will build is also included as [example](https://github.com/TheThingsNetwork/java-app-lib/blob/master/sample/src/main/java/org/thethingsnetwork/java/app/sample/Test.java) in the Java library.
+
+## Setup
+Let's install Java, create a Java project and require the TTN Client.
+
+1.  [Download](https://www.java.com/fr/download/) and install Java. Under debian-based systems, use :
+
+    ```bash
+    sudo apt-get install openjdk-8-jdk
+    ```
+2.  [Download](https://maven.apache.org/download.cgi) and install maven. Under debian-based systems, use :
+
+    ```bash
+    sudo apt-get install maven
```

Comment:
Maybe for Windows link https://www.mkyong.com/maven/how-to-install-maven-in-windows/